edit.go

```diff
@@ -127,6 +139,45 @@ func variableMenu() {
 	color.Unset()
 }
 
+func (e *Edit) importCredsFromEnv() (*vaulted.AWSCredentials, error) {
+	creds := e.swipeCredsFromEnv()
+	if creds != nil {
+		for {
+			warningColor.Println("There appear to be AWS credentials in your current environment.")
+			input, err := e.readPrompt("Would you like to import these credentials? (Y/n): ")
+			if err != nil {
+				return nil, err
+			}
+
+			switch strings.ToLower(input) {
+			case "", "y", "yes":
+				return creds, nil
+			case "n", "no":
+				return nil, nil
+			default:
+				output("")
+				color.Red("Response not recognized. Please enter 'y' or 'n'.")
+				output("")
+			}
+		}
+	}
+
+	return nil, nil
+}
+
+func (e *Edit) swipeCredsFromEnv() *vaulted.AWSCredentials {
+	if os.Getenv("AWS_ACCESS_KEY_ID") != "" && os.Getenv("AWS_SECRET_ACCESS_KEY") != "" {
+		if os.Getenv("AWS_SESSION_TOKEN") == "" && os.Getenv("AWS_SECURITY_TOKEN") == "" {
```

Comment:
If the user has asked us to import creds and we elect not to because they appear to be STS creds, perhaps we should indicate to the user why we did not follow through with the request.